backend/strapi-sync/create-multiple-item-campgrounds.js

```diff
@@ -425,14 +425,12 @@ async function createCampground(item) {
   const campground = await createModel(Campground, data);
 
   await Promise.all(
-    item.items.map(
-      async (feature) => await updateFeature(feature, campground.id),
-    ),
+    item.items.map(async (feature) => updateFeature(feature, campground.id)),
```

Comment:
You won't need the `async` on these lines either. The `async` keyword makes a function return a promise, but `updateFeature` already returns a promise so we can remove here.